pandas/tests/io/test_feather.py

```diff
@@ -1,7 +1,6 @@
 """ test feather-format compat """
-from distutils.version import LooseVersion
-
 import numpy as np
+from pkg_resources import parse_version
```

Comment:
Does this make setuptools a test dependency, rather than just a build dependency?

Reply:
Can't this be the same `Version` that is used everywhere else?

Reply:
Initially, I was going to use `pkg_resources`. But then I found that `pkg_resources.parse_version` is just a wrapper around `packaging.version.Version`. I forgot to change this one.

And we also use `pkg_resources` in https://github.com/pandas-dev/pandas/blob/d2881eb9105e114408d1e45d0326dd94fe1cb794/pandas/plotting/_core.py#L1749

So `setuptools` is technically an optional dependency as well.